pyproject.toml

```
@@ -31,68 +35,104 @@ classifiers = [
     "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
     "Topic :: Internet :: WWW/HTTP",
 ]
-requires = [
-    "starlette ==0.13.6",
-    "pydantic >=1.0.0,<2.0.0"
-]
-description-file = "README.md"
-requires-python = ">=3.6"
 
-[tool.flit.metadata.urls]
-Documentation = "https://fastapi.tiangolo.com/"
+[tool.poetry.dependencies]
+python = "^3.6"
+starlette = "0.13.6"
+pydantic = "^1.0.0"
+
+requests = { version = ">=2.24.0,<3.0.0", extras = ["all"] }
```

I think this should be
```suggestion
requests = { version = ">=2.24.0,<3.0.0", optional = true }
```

And the same change for all optional dependencies in `[tool.poetry.dependencies]` and `[tool.poetry.dev-dependencies]`

---

To clarify: in the current state this line would mean requiring `requests[all]` but we dont want that. We want to install requests given `fastapi[all]`

---

Thanks. I'll apply later. :)

---

Applied.